x/data/features/msg_anchor.feature

```diff
@@ -0,0 +1,21 @@
+Feature: MsgAnchor
+
+  Scenario Outline: sender must be valid
```

Comment:
we probably don't need this for every message, but it makes sense and is clear

Reply:
duplicate